README.md

```diff
@@ -0,0 +1,91 @@
+# pipsi
+
+pipsi = **pip** **s**cript **i**nstaller
+
+## What does it do?
+**pipsi** makes installing python packages with global entry points painless. These are Python packages that expose an entry point through the command line such as [black](https://pypi.org/project/black/), [pre-commit](https://pypi.org/project/pre_commit/), [Pygments](https://pypi.org/project/Pygments/), [poetry](https://pypi.org/project/poetry/), and many others.
```

Comment:
This might seem like an endorsement of any of those packages. Why do we need examples anyway? We are referring to almost any Python CLI app.

Reply:
Using an example of a package one would use changes it from an abstract description to a concrete one. IMO examples are the quickest way to convey how software works. But agreed, don't want to appear to be endorsing projects.

If you think examples are useful, maybe we can suggest with a non-controversial package? I think Pygments was already referred to in the original description so maybe I can just leave at that.